pkg/kudoctl/cmd/testdata/plans-list.golden

```diff
@@ -0,0 +1,11 @@
+plans
```

Comment:
I like golden files but this seems to be too small to be one. wdyt about putting it into a multiline var in the test above? Same for the other two golden files below.

Reply:
I actually like this one a lot.   it is a great output example that could be referenced via documentation (when using asciidoc).  It is also easy for a developer to "see" what the output is intended to be.

Reply:
Hm, it's not exactly YAML files so that they absolutely must be referenced to stay consistent with the codebase. And a developer can "see" them even better if they're in the same test file, where they're used. But I don't feel too strongly about it so if you insist 🤷‍♂